docs/plugins/ingest-useragent.asciidoc

```diff
@@ -0,0 +1,66 @@
+[[ingest-useragent]]
+=== Ingest Useragent Processor Plugin
+
+The Useragent processor extracts details from the useragent string a browser sends with its web requests. 
```

Comment:
I see 01cffd413d90250c19bfef66ccdfd33882d58283 but I meant everywhere, not just in the docs.